src/Protos/VerifiableEventStore.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "Primitives.proto";
+
+option csharp_namespace = "ProjectOrigin.VerifiableEventStore.Models";
+
+package ProjectOrigin.VerifiableEventStore.v1;
+
+message Event {
+    EventId Id = 1;
+    bytes Content = 2;
+}
+
+message EventId {
+    ProjectOrigin.Registry.v1.Uuid EventStreamId = 1;
+    uint32 Index = 2;
+}
+
+message MerkleProof {
+    Event Event = 1;
+    RootReference BlockchainReference = 2;
+    sint32 LeafIndex = 3;
```

Comment:
Is this the sub-proof that a user recieves for a transaction - after the batch is validated as a merkleproof, that can be proven valid externally by using the root-hash and the sub-proof, ie proof of uniqueness for the transaction ?